tools/2025-01-gitops-technical-design.md

```diff
@@ -129,9 +103,11 @@ components interact with each other in the context of this proposal.
 Include separate high level architecture diagram and component specific diagrams, wherever appropriate.
 -->
 
-!Architecture Diagram
+![Architecture Diagram Part 1](2025-01-gitops-technical-design/rfc-1.png)
```

Comment:
are there changes needed to be made to the Flux codebase for detecting `*.bicep` file changes, or can this be done through configuration?

Reply:
We handle this in the controller code by detecting the `radius-gitops-config.yaml`